From a39abaf1d2ada58ee3ec7805ddf4384e4f5639b0 Mon Sep 17 00:00:00 2001
From: Luo Jie <luoj@qti.qualcomm.com>
Date: Mon, 8 Jun 2026 14:35:08 +0800
Subject: [PATCH] driver: clk: ipq-cmn-pll: Remove eth-25mhz output clock of
 IPQ9650

The eth-pon clock (clk_eth_pon_ops) already manages CLK25M_EN_BIT and
CLK31P25M_EN_BIT through its mux-aware enable/disable logic, so eth-25mhz was a redundant and conflicting gate on the same hardware bit.

Change-Id: I7d4470803086dc7fdfe4ee46a6f355ad568696d2
Signed-off-by: Luo Jie <luoj@qti.qualcomm.com>
---
 drivers/clk/qcom/ipq-cmn-pll.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/clk/qcom/ipq-cmn-pll.c b/drivers/clk/qcom/ipq-cmn-pll.c
index e06aac7a1e39..dc1fd3f78c04 100644
--- a/drivers/clk/qcom/ipq-cmn-pll.c
+++ b/drivers/clk/qcom/ipq-cmn-pll.c
@@ -241,7 +241,6 @@ static const struct cmn_pll_fixed_output_clk ipq9650_output_clks[] = {
 	CLK_PLL_GATE(IPQ9650_ETH0_50MHZ_CLK, "eth0-50mhz", 50000000, CLK50M_EN_BIT),
 	CLK_PLL_GATE(IPQ9650_ETH1_50MHZ_CLK, "eth1-50mhz", 50000000, CLK50M_EN_BIT2_BIT),
 	CLK_PLL_GATE(IPQ9650_ETH2_50MHZ_CLK, "eth2-50mhz", 50000000, CLK50M_EN_BIT3_BIT),
-	CLK_PLL_GATE(IPQ9650_ETH_25MHZ_CLK, "eth-25mhz", 25000000, CLK25M_EN_BIT),
 	{ /* Sentinel */ }
 };
 
-- 
2.34.1

